CHANGES.md

```diff
@@ -1,3 +1,66 @@
+## edge-19.4.5
+
+**Significant Update**
+
+As of this edge release the proxy auto-injector component is always installed.
+To have the auto-injector inject a pod you still have the option to manually
```

Comment:
_To have the proxy injector auto-injects a pod_

Reply:
Also, _still have the option_ sounds optional. For auto-inject to work, that annotation must be added. How about _you can manually add the_?